wire-grpc-client/src/main/java/com/squareup/wire/GrpcClient.kt

```diff
@@ -44,9 +45,7 @@ class GrpcClient private constructor(
 ) {
   fun <T : Service> create(service: KClass<T>): T {
     val methodToService: Map<Method, GrpcMethod<*, *>> =
-        service.java.methods.associate { method ->
-          method to method.toGrpc<Any, Any>()
-        }
+        service.java.methods.associate { method -> method to method.toGrpc<Any, Any>() }
```

Comment:
`associateWith`

Reply:
was looking for this! It appears `associateWith` is on `Iterable`s but `methods` is an array here.

Reply:
Ah, that's a shame